lib/mail/version_specific/ruby_1_9.rb

```diff
@@ -68,7 +68,7 @@ def Ruby19.q_value_decode(str)
       match = str.match(/\=\?(.+)?\?[Qq]\?(.+)?\?\=/m)
       if match
         encoding = match[1]
-        str = Encodings::QuotedPrintable.decode(match[2].gsub(/_/, '=20'))
+        str = Encodings::QuotedPrintable.decode(match[2].gsub(/_/, '=20').gsub(/\=$/, ''))
```

Comment:
What is this backporting?

Can you cherry-pick the backported commit directly, preserving the commit message, author, spec, etc?